src/pages/DataAccessRequestApplication.js

```diff
@@ -481,43 +556,87 @@ class DataAccessRequestApplication extends Component {
     this.setState({ showDialogSave: true });
   };
 
-  dialogHandlerSubmit = (answer) => (e) => {
+  //Can't do uploads in parallel since endpoints are post and they both alter attributes in json column
+  //If done in parallel, updated attribute of one document will be overwritten by the outdated value on the other
+  saveDULDocuments = async(uploadedIrbDocument = null, uploadedCollaborationLetter = null, referenceId) => {
```

Comment:
Similar naming nit. We use `DUL` to refer to data use letter documents which are usually a hard-copy (pdf, word, etc.) of the data use restrictions on the dataset. These are provided by the study's PI or PI's institution directly to our Office of Research Subject Protection.